todo_app/app.py

```diff
@@ -97,43 +111,64 @@ def login_callback():
         name = github_user['login']
         user = User(name)
         login_user(user)
+        app.logger.info("User %s logged in" % name)
         return redirect(url_for('index'))
 
     @app.route('/add_item', methods=['POST'])
     @check_writer_role
     @login_required
     def add_todo_item():
-        item_title = request.form.get('title')
-        item_description = request.form.get('description')
-        mongo.add_item(item_title, item_description)
+        try:
+            item_title = request.form.get('title')
+            item_description = request.form.get('description')
+            mongo.add_item(item_title, item_description)
+            app.logger.info("Added todo item %s" % request.form)
```

Comment:
You could log the id (you can get the id of the inserted document from the return value of the insert method)